gammapy/time/lightcurve.py

```diff
@@ -68,3 +69,35 @@ def simulate_example(cls):
         lc['FLUX_ERR'] = Quantity([0.1, 0.4, 0.7, 0.9], 'cm^-2 s^-1')
 
         return lc
+
+    def compute_fvar(self):
+        """Calculate the fractional excess variance (Fvar) of `LightCurve`.
+
+        """
+        time_min = self['TIME_MIN']
+        time_max = self['TIME_MAX']
```

Comment:
Looks like you have mixed tabs and spaces?
Should all be spaces and 4 space indent.

pep8 tool will tell you about it, autopep8 tool can auto-fix it.
(or configure your editor for Python dev)